tools/marvin/setup.py

```diff
@@ -27,7 +27,7 @@
         raise RuntimeError("python setuptools is required to build Marvin")
 
 
-VERSION = "4.20.0.0-SNAPSHOT"
+VERSION = "4.20.0.0"
```

Comment:
needed?

Reply:
may be not needed, but not an issue, as `mvn-setup.py` will change it to `4.20.0.0`
let me revert it
```suggestion
VERSION = "4.20.0.0-SNAPSHOT"
```